sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -327,13 +340,18 @@ ng_ipv6_addr_t *ng_ipv6_netif_find_addr(kernel_pid_t pid, const ng_ipv6_addr_t *
 }
 
 static uint8_t _find_by_prefix_unsafe(ng_ipv6_addr_t **res, ng_ipv6_netif_t *iface,
-                                      const ng_ipv6_addr_t *addr, bool src_search)
+                                      const ng_ipv6_addr_t *addr, bool src_search,
```

Comment:
Since `only` is only used for source address determination the `src_search` argument is now not needed anymore.